### PR TITLE
fix crash when walk up sites are not available

### PR DESCRIPTION
**Pull request by ai2160**
When walk up site (site-tag 'W') is unavilable, we try to fetch non existing 'href' element. For all 'available' codes, we must ensure href is available before fetching it.

Also fixed incorrect site_url for 'call only' sites.

### CODE REVIEW

**Comment by schlosser** [comment]
Great fix!  Just remove your email address and I'll merge!

**Comment by ai2160** [comment]
Removed changes to config.json.

File: scraper.py [hunk 1]
```diff
@@ -111,7 +135,7 @@
 inlined_html = h.unescape(response.text)
 
 requests.post(MG_URL, auth=('api', MG_KEY), data={
-    'from': '"Yosemite Campsite Scraper" <yosemite@schlosser.io>',
+    'from': '"Campsite Scraper" <camper@westerncamperphoto.com>',
```
Comment (schlosser) [inline scraper.py:138]: Please remove this change

File: generate_config.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
```
Comment (schlosser) [inline generate_config.py:1]: Please remove this entire file

File: config.json [hunk 3]
```diff
@@ -1,25 +1,59 @@
...
-			"length": 3
-		},
-		{
-			"start_date": "07/31/2015",
-			"length": 3
-		}
-	]
-}
\ No newline at end of file
+    "emails": [
```
Comment (schlosser) [inline config.json:2]: Please remove all changes to this file

**Comment by schlosser** [comment]
Hello, thanks for the above changes, but would you please remove all changes to `config.json` and untrack your `generate_config.py` file, as it is for your specific use case?  Thanks.

**Comment by ai2160** [comment]
Hi, I plan to refactor and make it more generic so it can be used on recreation.gov. Closing this PR.

**Pull request closed without merging**